test/matter/interaction/InteractionProtocolTest.ts

```diff
@@ -51,6 +56,42 @@ const READ_RESPONSE: DataReport = {
     ]
 };
 
+const WRITE_REQUEST: WriteRequest = {
+    interactionModelRevision: 1,
+    suppressResponse: false,
+    timedRequest: false,
+    writeRequests: [
+        {
+            path: { endpointId: 0, clusterId: 0x28, attributeId: 100 },
+            data: TlvUInt8.encodeTlv(3),
+            dataVersion: 0,
+        },
+        {
+            path: { endpointId: 0, clusterId: 0x28, attributeId: 5 },
+            data: TlvString.encodeTlv("test"),
+            dataVersion: 0,
+        },
+    ],
+    moreChunkedMessages: false,
+};
+
+const WRITE_RESPONSE: WriteResponse = {
+    interactionModelRevision: 1,
+    writeResponses: [
+        {
+            path: {
```

Comment:
nit: match the declaration above so it is clear which path this is:
path: { endpointId: 0, clusterId: 0x28, attributeId: 100 },